internal/cli/kraft/net/create/create.go

```diff
@@ -20,6 +20,13 @@ import (
 	"kraftkit.sh/machine/network"
 )
 
+const (
+	// defaultNetworkMaxSize is the maximum size of a network name on Linux.
+	// We subtract 5 to account for the network interface suffix '@if%d%d%d'.
+	// The value 16 is taken from the glibc headers.
+	defaultNetworkNameMaxSize = 15 - 5
```

Comment:
This should be internalised to the bridge network implementation, since it is specific to Linux.

Reply:
done

Reply:
Perfect, thanks!